### PR TITLE
Config parameter

### PR DESCRIPTION
**Pull request by micahkendall**
Remade pull request without forked repo.

### CODE REVIEW

**Comment by micahkendall** [comment]
Using Ops.Applicative now, also new Parameters.hs file. Using FilePath as suggested.

**Comment by micahkendall** [comment]
Added helper too.

**Comment by samuelWilliams99** [comment]
We can't merge this until build passes, which means running hlint and fourmolu. We can either get the make file setup first then run it here, or you can find the commands directly and run those on this branch

File: src/Parameters.hs
Comment (samuelWilliams99) [inline src/Parameters.hs:7]: Need explicit imports here

File: src/Parameters.hs
Comment (samuelWilliams99) [inline src/Parameters.hs:27]: I think you can lose this option

**Review by samuelWilliams99 — COMMENTED** [review]
Looks good! Once passing CI, we're good to merge

File: src/Parameters.hs
Comment (t4ccer) [reply src/Parameters.hs:7]: We are missing `-Wmissing-import-lists` in `cabal` file

Comment (samuelWilliams99) [reply src/Parameters.hs:7]: Yep, lets get that in there

File: src/Parameters.hs
Comment (ryukzak) [inline src/Parameters.hs:18]: Maybe mconcat from Data.Monoid  will be better.

Comment (micahkendall) [reply src/Parameters.hs:18]: I think the mappend operator is more idiomatic in this case.

**Review by ryukzak — APPROVED** [review]

**Review by samuelWilliams99 — APPROVED** [review]
Looks good!

**Review by t4ccer — APPROVED** [review]

**Pull request merged**
